apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -679,7 +685,8 @@ class TopInstructions extends Component {
             </div>
             {/* For CSF contained levels we use the same collapse function as CSD/CSP*/}
             {!this.props.isEmbedView &&
-              (isCSDorCSP || this.props.hasContainedLevels) && (
+              (isCSDorCSP || this.props.hasContainedLevels) &&
+              false && (
```

Comment:
hmmm will this affect levels that are not AILab and currently out in the wild?

Reply:
Good catch; that was temporary and is now addressed.